trigger-openqa_in_openqa

```diff
@@ -22,12 +22,20 @@ build_tag=${BUILD_TAG:-}
 client_prefix=${client_prefix:-}
 full_run=${FULL:-}
 group_id="${group_id:-"openQA"}"
+osc=${osc:-osc}
+src_project=${src_project:-devel:openQA}
+staging_project=${staging_project:-devel:openQA:testing}
+dst_project=${dst_project:-devel:openQA:tested}
```

Comment:
Last nitpick because I like how you solved it in [os-autoinst-obs-auto-submit](https://github.com/os-autoinst/scripts/pull/352/files#diff-e9e4a2f7fd0feb7816f56d2fcd9ad02870fc9344ea52dfc275699c1a7d2cbff8) and I think it should behave in the same way, when I only override `$src_project`.

```suggestion
staging_project=${staging_project:-${src_project}:testing}
dst_project=${dst_project:-${src_project}:tested}
```